src/assets/scss/_about-me.scss

```diff
@@ -0,0 +1,190 @@
+//About me section
+
+.about-me {
+    height: 45rem;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    z-index: 5;
+
+    @media (max-width: $mobile-size) {
+      height: 20rem;
+    }
+       
+    &__text {
+      flex-direction: column;
+      width: 50rem;
+      padding-top: 12rem;
+      font-weight: 300;
```

Comment:
font-weight najlepiej dodać do zmiennej :) Będziesz miała light, normal i bold, a jak zmieniałabys font i potrzebowa trochę innych wartości to zmiana by była w 1 miejscu :)